base/rational.jl

```diff
@@ -375,8 +375,8 @@ function ^(x::Rational, n::Integer)
 end
 
 ^(x::Number, y::Rational) = x^(y.num/y.den)
-^{T<:AbstractFloat}(x::T, y::Rational) = x^(convert(T, y.num / y.den))
-^{T<:AbstractFloat}(x::Complex{T}, y::Rational) = x^(convert(T, y.num / y.den))
+^{T<:AbstractFloat}(x::T, y::Rational) = x^(convert(T,y.num)/convert(T,y.den))
+^{T<:AbstractFloat}(x::Complex{T}, y::Rational) = x^(convert(T,y.num)/convert(T,y.den))
```

Comment:
It will dispatch to this method:
https://github.com/JuliaLang/julia/blob/b50604121db9cc6a7b1294ca040fc9a715c00359/base/rational.jl#L71-L74
which actually will be better, as it will handle, e.g. `^(x::Float64, y::Rational{BigInt})` by promoting to `BigFloat`.